projects/jupyter-server-ydoc-codio/Untitled.ipynb

```
@@ -0,0 +1,55 @@
+{
```

Flake8 (7.2.0)

[error] undefined name 'qwef'

(F821)